src/test/java/com/josdem/vetlog/service/RestServiceTest.java

```diff
@@ -50,7 +51,7 @@ void setup() {
     @DisplayName("sending message")
     void shouldSendMessage(TestInfo testInfo) throws IOException {
         log.info("Running: {}", testInfo.getDisplayName());
-        RestService restService = mock(RestService.class);
+        RestService restService = Mockito.mock(RestService.class);
```

Comment:
redundant change

Reply:
> redundant change

Agreed. Good catch! Not sure how that was intruduced I am assumming `spotlessApply` did it.

Reply:
Fixed here: https://github.com/josdem/vetlog-spring-boot/pull/287/commits/33d77c98a94d9adcca7978d64df954dbda1fc611